doc/deimos.rst

```diff
@@ -12,26 +12,21 @@ settings that are related to the Keck/DEIMOS spectrograph.
 Deviations
 ==========
 
-Here are the deviations from the default settings
-for DEIMOS (set in the settings.keck_deimos file)::
-
-    settings trace slits sigdetect 50.0
-    settings trace slits number -1
-    settings trace slits tilts params 1,1,1
-    settings trace slits tilts method spca
-    settings trace slits pca params [3,2,1,0]
-    settings trace slits polyorder  3
-    settings trace slits sobel mode nearest
-    settings trace slits fracignore  0.02   # 0.02 removes star boxes of 40pix size or less (and any real ones too!)
-    settings bias useframe overscan
-    settings pixelflat combine method median
-    settings pixelflat combine reject level [10.0,10.0]
+Here are the main deviations from the default settings
+for DEIMOS (see `default_pypeit_par`in
+the spectrographs.keck_deimos.py file)::
```

Comment:
You can include direct links to the function in the docs.  Change to

```
(see :func:`~pypeit.spectrographs.keck_deimos.KeckDEIMOSSpectrograph.default_pypeit_par`)
```

Reply:
modified